invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -433,8 +434,9 @@ class PointSchema(Schema):
 class LocationSchema(Schema):
     """Location schema."""
 
-    point = fields.Nested(PointSchema)
+    geometry = fields.Nested(GeoJSONSchema, required=True)
     place = SanitizedUnicode(required=True)
+    identifiers = fields.Dict()
```

Comment:
Question: We don't have a schema for identifers that ensures that both keys and values are strings?

Reply:
This is one of the point raised when developing the data model (cc @zzacharo). We have identifiers such as:

```
{
     identifiers: value,
     scheme: value
}
```

But also as:

```
{
     scheme-value: identifier-value
}
```

The second one was some of the ones than among others could cause the mapping explosion. But it was decided that there are not enough key possibilities for this to happen.

I understood from @zzacharo that when you two discussed about this (related to frontend) was decided that there was not gonna be any change in the data model.

As for a specific schema for this case (the second of the examples above), a Marshmallow schema would be complex because the key is unknown and therefore validation would be complex (and not much more than what we obtain with `Dict`. As for the first example, there is a schema and it is used when appropriate.

Reply:
Ok. Then what we would need is some sort of identifier validator (on e.g. the Dict field (

```
identifiers = fields.Dict(validate=IdentifiersValidator(...))
```